polypomo

```diff
@@ -133,6 +138,33 @@ class Status:
             self.timer = Timer(self.worktime)
 
 
+@contextmanager
+def setup_conn(path):
+    # check if the database exist
+    conn = sqlite3.connect(path)
+    cur = conn.cursor()
+    cur.execute("""CREATE TABLE IF NOT EXISTS sessions (
+                        date text,
+                        start text,
+                        stop text
+        )""")
+    try:
+        yield cur
+    finally:
+        conn.commit()
+        conn.close()
+
+def db_save(status, path, active):
```

Comment:
From the arguments this looks like it could be a member of `Status`.
I agree that conceptually its not the right class but it will simplify the function signature.

Can also be renamed to `save()` given we only have one way of saving at the moment.